jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml

```diff
@@ -2,28 +2,41 @@
   <!-- mainly copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
   <arg name="enable_auto_dock" default="true" />
   <arg name="joy_device" default="/dev/ps3joy"/>
-  <arg name="disable_arm_teleop" default="true"/>
 
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
   <node name="joy" pkg="joy" type="joy_node" respawn="true">
     <param name="autorepeat_rate" value="1"/>
     <param name="dev" value="$(arg joy_device)"/>
   </node>
 
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
+  <!-- copied to support unsafe teleop. -->
+  <!-- copied for button mapping is changed to standardize with PR2 joy. -->
   <node name="teleop" pkg="fetch_teleop" type="joystick_teleop" respawn="true">
     <remap from="teleop/cmd_vel" to="/teleop/cmd_vel/unsafe" />
-    <!-- set these two motions as non-existent button 17 -->
-    <!-- from fetch_teleop version 0.7.11 (https://github.com/fetchrobotics/fetch_ros/pull/61), arm teleop is added and it is necessary to update jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml -->
-    <param name="arm/button_arm_linear" value="17" if="$(arg disable_arm_teleop)"/>
-    <param name="arm/button_arm_angular" value="17" if="$(arg disable_arm_teleop)"/>
+    <param name="arm/button_arm_linear" value="4"/>
+    <param name="arm/button_arm_angular" value="6"/>
+    <param name="head/button_deadman" value="11"/>
     <param name="base/use_mux" value="false" />
   </node>
 
-  <node name="cmd_vel_mux" pkg="topic_tools" type="mux" respawn="true"
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
+  <node name="cmd_vel_mux" pkg="topic_tools" type="mux" respawn="true" output="screen"
 	args="base_controller/command /cmd_vel /teleop/cmd_vel">
     <remap from="mux" to="cmd_vel_mux" />
   </node>
 
-  <node name="controller_reset" pkg="fetch_bringup" type="controller_reset.py" />
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
+  <!-- copied for button mapping is changed to standardize with PR2 joy. -->
```

Comment:
need to launch controller_reset because of launch_teleop:=false is set at /etc/ros/indigo/robot.launch, see fetchrobotics/fetch_robots#42

Reply:
I added the comment.